### PR TITLE
Created GET /categories/{id} Route 

### PR DESCRIPTION
**Pull request by seoulstice**
### Description
* Created happy and sad path tests for GET /categories/{id} endpoint
* Created API endpoint for GET /categories/{id}
* Created get_category(id) method

### Type of change
* New feature (non-breaking change which adds functionality)

Closes Issue #198

### CODE REVIEW

File: tests/unit/test_routes.py [hunk 1]
```diff
@@ -246,6 +246,33 @@ def test_categories(module_client, module_db):
...
+    client = module_client
+
+    response = client.get('api/v1/categories/4')
+
+    # Status should be OK
+    assert (response.status_code == 200)
+
+    category = response.json['data']
+    assert (isinstance(category.get('name'), str))
+    assert (category.get('name') == 'Tutorials')
```
Comment (aaron-suarez) [inline tests/unit/test_routes.py]: We don't need this assertion, it too tightly couples the data with the test. If we were to change the yaml file, for example, `Tutorials` might not end up as the fourth category. Just checking that it's a string with ID 4 is fine

**Comment by aaron-suarez** [comment]
Can we please get one more check in addition to the `too_low` and `too_high` that verifies a 404 when you send a string of non-numeric characters? I think we used `waffles` in another test, but it would be nice to take this opportunity to get some strings with other ascii characters and see how it behaves in that situation.

**Comment by aaron-suarez** [comment]
Other than that, I'm good with the change! Thanks so much for the contribution! Let's make it happen

**Comment by seoulstice** [comment]
Made the suggested changes and thanks for the welcome to OC.  This is my first time contributing to open source and my first time working with Python/Docker 👍

**Pull request merged**